doc/sphinx/source/graphdatascience.rst

```diff
@@ -3,4 +3,5 @@ GraphDataScience
 
 .. autoclass:: graphdatascience.GraphDataScience
     :members:
-    :inherited-members:
\ No newline at end of file
+    :inherited-members:
+    :exclude-members: __init__
```

Comment:
The class itself already gets the __init__ params documentation, so if we don't exclude __init__ like this they are duplicated